index.html

```diff
@@ -89,16 +89,16 @@
             class="rec-7"
```

Comment:
Kindly, I suggest you use the semantic NAV tag for your main menu (UL/LI), the rest (like the logo) you should leave it out, and use HEADER to wrap the entire top bar

Reply:
Kindly, all the design images of your site are placed in the background, I suggest you use background-image

Reply:
This isn't the requirement for this pull request